runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPanel.java

```diff
@@ -341,8 +341,7 @@ void add(final String eventName, final int actorLevel, LootTrackerItem[] items)
 		LootTrackerBox box = buildBox(record);
 		if (box != null)
 		{
-			box.rebuild();
-			updateOverall();
+			rebuild();
```

Comment:
This seems much more expensive now, we've had issues in the past with perf. issues on the loottracker due to too much rebuilding. @abextm do you know if this is ok?

Reply:
Yeah, doing this will reopen #6134